tortoise/verifying_tortoise.go

```diff
@@ -130,19 +241,23 @@ func blockIDsToString(input []types.BlockID) string {
 	return str
 }
 
-// TODO: cache but somehow check for changes (hare that didn't finish finishes..) maybe check hash?
-func (t *turtle) getSingleInputVectorFromDB(lyrid types.LayerID, blockid types.BlockID) (vec, error) {
-	if !lyrid.After(types.GetEffectiveGenesis()) {
+// returns the binary local opinion on the validity of a block in a layer (support or against).
+// abstain is not an option here: if the current opinion on a layer is abstain, an opinion on
+// a block in that layer will be against.
+// TODO: does this ever need to return abstain?
```

Comment:
if we know hare wasn't terminated yet we should return abstain, this is why the error case is here. I assumed a terminated hare with empty set is perfectly valid so no err but if there are no results return an error to indicate it's still running.

Reply:
fixed this, thanks for highlighting